modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,428 @@
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" ${PUBLIC_API_ENDPOINT}/v1beta1/clusters/$CLUSTER_ID
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to enable PSC.
++
+Set the following environment variable:
++
+--
+`ACCEPT_LIST`: a JSON list specifying the projects from which incoming connections will be accepted. All other sources. For example, `[{"source": "consumer-project-ID-1"},{"source": "consumer-project-ID-2"}]`.
+--
++
+[,bash]
+----
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+    "private_link": {
+        "enabled": true,
+        "gcp": {
+            "consumer_accept_list": $ACCEPT_LIST
+        }
+    }
+}
+EOF`
+curl -v -X PATCH \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" ${PUBLIC_API_ENDPOINT}/v1beta1/clusters/$CLUSTER_ID
+----
++
+Wait for the cluster to apply the new configuration (around 15 minutes). The PSC service attachment is available when the cluster update is complete. You can monitor the service attachment creation by running the following `gcloud` command:
++
+[,bash]
+----
+gcloud compute service-attachments list --project '<service-project-ID>'
+----
+
+== Test the connection
+
+You can test the PSC connection from any VM or container in the consumer VPC. If configuring a client isn't possible right away, you can do these checks using `rpk` or curl:
+
+=== Test Kafka API
+
+. Set the following environment variables.
++
+[,bash]
+----
+REDPANDA_BROKERS='<PSC-Kafka-API-seed-URL>'
+REDPANDA_SASL_MECHANISM='<SCRAM-SHA-256-or-SCRAM-SHA-512>'
+REDPANDA_SASL_USERNAME='<username>'
+REDPANDA_SASL_PASSWORD='<password>'
+----
+
+. Check the cluster status using `rpk`.
++
+[,bash]
+----
+rpk cluster info
+----
+
+. Create a test topic.
++
+[,bash]
+----
+rpk topic create psc-test
+----
+
+. Produce to the test topic.
++
+[tabs]
+====
+rpk::
++
+[,bash]
+----
+echo 'hello PSC!' | rpk topic produce psc-test
+----
+Curl::
++
+[,bash]
+----
+curl -s \
+  -X POST \
+  <PSC-HTTP-Proxy-seed-URL>/topics/psc-test \
+  -H "Content-Type: application/vnd.kafka.json.v2+json" \
+  -d '{
+  "records":[
+      {
+          "value":"hello PSC!"
+      }
+  ]
+}'
+----
+====
+
+. Consume from the test topic.
++
+[tabs]
+====
+rpk::
++
+[,bash]
+----
+rpk topic consume psc-test -n 1
+----
+Curl::
++
+--
+. Create a consumer.
++
+[,bash]
+----
+curl -s \
+  -X POST \
+  "<PSC-HTTP-Proxy-seed-URL>/consumers/test_group"\
+  -H "Content-Type: application/vnd.kafka.v2+json" \
+  -d '{
+  "format":"json",
+  "name":"test_consumer",
+  "auto.offset.reset":"earliest",
+  "auto.commit.enable":"false",
+  "fetch.min.bytes": "1",
+  "consumer.request.timeout.ms": "10000"
+}'
+----
+. Subscribe the consumer to the test topic.
++
+[,bash]
+----
+curl -s -o /dev/null -w "%{http_code}" \
+  -X POST \
+  "<PSC-HTTP-Proxy-seed-URL>/test_group/instances/<instance>/subscription"\
```

Comment:
The [tutorial page](https://docs.redpanda.com/current/develop/http-proxy/#subscribe-to-the-topic) only provides an HTTP proxy URL of `<base-uri>/subscription` whereas the [reference doc](https://docs.redpanda.com/api/pandaproxy-rest/#post-/consumers/-group_name-/instances/-instance-/subscription) specifies the following: `/consumers/{group_name}/instances/{instance}/subscription` What should go in the `instance` path parameter?

Reply:
The instance ID returned in the response of [POST /consumers/{group_name}](https://docs.redpanda.com/api/pandaproxy-rest/#post-/consumers/-group_name-):
```{
"instance_id": "string",
"base_uri": "string"
}```